package.json

```diff
@@ -16,12 +16,13 @@
   "author": "Apostrophe Technologies",
   "license": "MIT",
   "devDependencies": {
+    "apostrophe": "^3.0.1",
     "eslint": "^7.9.0",
     "eslint-config-apostrophe": "^3.4.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1"
+    "mocha": "^7.1.2"
+  },
+  "dependencies": {
+    "common-tags": "^1.8.0",
+    "dayjs": "^1.10.5"
```

Comment:
All we're doing with this is generating and formatting a `today` date. Can we drop this dep?